schemas/alpha/azure.yaml.json

```diff
@@ -343,6 +343,59 @@
                     ]
                 }
             }
+        },
+        "state": {
+            "type": "object",
+            "title": "The state configuration used for the project.",
+            "description": "Optional. Provides additional configuration for state management.",
+            "additionalProperties": false,
+            "properties": {
+                "remote": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "title": "The remote state configuration.",
+                    "description": "Optional. Provides additional configuration for remote state management such as Azure Blob Storage.",
+                    "required": [
+                        "backend"
+                    ],
+                    "properties": {
+                        "backend": {
+                            "type": "string",
+                            "title": "The remote state backend type.",
+                            "description": "Optional. The remote state backend type. (Default: AzureStorage)",
```

Comment:
Since we may add Azure File Storage in the future, I think this name should be AzureBlobStorage.